Makefile

```diff
@@ -24,3 +24,22 @@ release:
 	mkdir bin
 	cp src/Adobe.AEPAssurance.Android/bin/Debug/*.nupkg ./bin
 	cp src/Adobe.AEPAssurance.iOS/bin/Debug/*.nupkg ./bin
+
+
+AEPASSURANCE_SDK_PATH = ./acp-sdk
```

Comment:
Other extension add acp-sdk/ to .gitignore just to be safe.
https://github.com/adobe/xamarin-acpanalytics/blob/master/.gitignore#L345

Reply:
will do